sources/tests/workflow-docker-compose.yml

```diff
@@ -0,0 +1,57 @@
+
+services:
+
+  zeebe-test:
+    image: camunda/zeebe:${CAMUNDA_PLATFORM_VERSION:-8.1.1}
+    container_name: zeebe-test
+    environment:
+      - ZEEBE_BROKER_EXPORTERS_ELASTICSEARCH_CLASSNAME=io.camunda.zeebe.exporter.ElasticsearchExporter
+      - ZEEBE_BROKER_EXPORTERS_ELASTICSEARCH_ARGS_URL=http://elasticsearch-test:9200
+      - ZEEBE_BROKER_EXPORTERS_ELASTICSEARCH_ARGS_BULK_SIZE=1
+      - ZEEBE_BROKER_DATA_DISKUSAGECOMMANDWATERMARK=0.998
+      - ZEEBE_BROKER_DATA_DISKUSAGEREPLICATIONWATERMARK=0.999
+      - "JAVA_TOOL_OPTIONS=-Xms256m -Xmx256m"
+    restart: always
+    networks:
+      - local-network-tests
+    depends_on:
+      - elasticsearch-test
+
+  elasticsearch-test:
```

Comment:
Just to double-check, removing the `elasticsearch-test` service and also the "depends_on: - elasticsearch-test (line 17)" generates an error while starting the container, right?

Reply:
I verified again and got the same error, but on investigating further I found that these variables must be used internally to connect elastic search and removed them.  After removing them, it worked without elastic search.

![image](https://user-images.githubusercontent.com/54600590/225162669-119a181e-6038-435a-8358-5fa33c6f80ab.png)